doc/stdenv/stdenv.chapter.md

```diff
@@ -1550,6 +1550,14 @@ sorry, unimplemented: __builtin_clear_padding not supported for variable length
 
 This flag adds the `-fstack-clash-protection` compiler option, which causes growth of a program's stack to access each successive page in order. This should force the guard page to be accessed and cause an attempt to "jump over" this guard page to crash.
 
+#### `pacret` {#pacret}
+
+This flag adds the `-mbranch-protection=pac-ret` compiler option on aarch64 linux targets. This uses ARM v8.3's Pointer Authentication feature to sign function return pointers before adding them to the stack. The pointer's authenticity is then validated before returning to its destination. This dramatically increases the difficulty of ROP exploitation techniques.
```

Comment:
Nit: `aarch64-linux` or “AArch64 Linux”.

Looks good other than that. This or https://github.com/NixOS/nixpkgs/pull/326819 will need rebasing after the other lands, right? Do you have a preference for which goes first?

Reply:
Nitted.

I have no preference in rebasing order.